content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -36,12 +36,12 @@ complete -o default -F __start_kubectl k
 ### ZSH
 
 ```bash
-source <(kubectl completion zsh)  # setup autocomplete in zsh into the current shell
+source <(kubectl completion zsh)  # set up autocomplete in zsh into the current shell
 echo '[[ $commands[kubectl] ]] && source <(kubectl completion zsh)' >> ~/.zshrc # add autocomplete permanently to your zsh shell
 ```
-### A Note on --all-namespaces
+### A Note on `--all-namespaces`
```

Comment:
Sentence case for headings in the page body. Also, I'm not sure if we need to include the --all-namespaces in this format tbh.
```suggestion
### A note on `--all-namespaces`
```

Reply:
Thanks. Changed now.